sideboard-development.ini.template

```diff
@@ -0,0 +1,26 @@
+debug = ${DEBUG}
+priority_plugins = "uber",
+default_url = "${DEFAULT_URL}"
```

Comment:
Based on what I have in my local file, the default_url and path is not needed for Sideboard.

Reply:
At least in my setup having default_url != path seems to cause an infinite redirect when loading /.

Reply:
Well then! Guess it's required after all!

Actually I wonder if that's why I couldn't switch my MAGFest config to the docker setup.